contracts/staking/ServiceStakingBase.sol

```diff
@@ -540,26 +635,36 @@ abstract contract ServiceStakingBase is ERC721TokenReceiver, IErrorsRegistries {
             revert OwnerOnly(msg.sender, sInfo.owner);
         }
 
-        // Check that the service has staked long enough, or if there are no rewards left
+        // Get the staking start time
+        // Note that if the service info exists, the service is staked or evicted, and thus start time is always valid
         uint256 tsStart = sInfo.tsStart;
+
+        // Check that the service has staked long enough, or if there are no rewards left
```

Comment:
we still need to discuss about the possible issue on code updates here

Reply:
Not sure I understand from description

Reply:
https://github.com/valory-xyz/autonolas-registries/pull/139#discussion_r1394762113